### ISSUE TITLE
Wrong result when adding any number to -0

### ISSUE THREAD

**Issue by Yayo-Arellano** [royNiladri/js-big-decimal#63]
**Describe the bug**
Wrong result when adding any number to -0

**To Reproduce**
```
 const n1 = new bigDecimal('-0');
 const n2 = new bigDecimal('10');
 const diff1 = n1.add(n2);
print(diff1.getValue()); // Will print -890
```

**Expected behavior**
The result should be 10

**Additional context**
If the operation is adding -0 to 10 the result will be correct.

**Comment by royNiladri** [comment]
This has been fixed in the latest version [1.3.13](https://www.npmjs.com/package/js-big-decimal/v/1.3.13)

**Comment by EmmanJayDizon** [comment]
thanks @royNiladri
